app/main.py

```diff
@@ -19,24 +16,25 @@ def __init__(self, distance_from_city_center: float, clean_power: int,
         self.count_of_ratings = count_of_rating
         self.to_wash = 0
 
-    def serve_cars(self, list_of_cars: list) -> float:
+    def serve_cars(self, list_of_cars: list[Car]) -> float:
         return sum([self.calculate_washing_price(car) for car in list_of_cars
                     if self.wash_single_car(car)])
 
-    def calculate_washing_price(self, other: typing.Any) -> float:
-        return round(other.comfort_class * (
-            self.clean_power - self.to_wash)
-            * self.average_rating / self.distance_from_city_center, 1)
+    def calculate_washing_price(self, other: Car) -> float:
+        return round(other.comfort_class
+                     * (self.clean_power - self.to_wash)
```

Comment:
Per our assignment "car's comfort class * difference between wash station's clean power and **_car's clean mark_* * * car wash station rating / car wash station distance to the center of the city", so self.to_wash -> other.clean_mark . 
While it probably will work withing **serve cars**, **calculate_washing_price** separately will give the wrong price.

Also, maybe other is not the best choice for an object name. Maybe just "car"?